lib/browser/api/app.js

```diff
@@ -24,8 +24,20 @@ Object.assign(app, {
     return Menu.getApplicationMenu()
   },
   commandLine: {
-    appendSwitch: bindings.appendSwitch,
-    appendArgument: bindings.appendArgument
+    appendSwitch() {
+      let castedArgs = [...arguments].map((arg) => {
```

Comment:
I think other places in the codebase usually convert `arguments` to an array by taking an `(...args)` parameter to the function.

Example: https://github.com/electron/electron/blob/d1274bb79f4aed25bec8c2a26dc0a8091f451a86/lib/browser/rpc-server.js#L186